### DISCUSSION TITLE
Latency Increase After Upgrading Next 12 to Next 13

### DISCUSSION THREAD

**Discussion by teddypiter** [vercel/next.js#55568] [Help]
### Summary

Last month, I upgraded my website from Next.js 12.1.6 to Next.js 13.4.19 and observed a significant increase in latency. Has anyone else experienced the same issue?

Are there any changes in Next.js 13 that could be affecting latency? Any help would be greatly appreciated!

![Screenshot 2023-09-19 123703](https://github.com/vercel/next.js/assets/53746400/582109cd-bec5-4724-9c22-a516a9395108)
 
The request throughput is also experiencing a significant decrease. Does this affect the latency increase?
![image](https://github.com/vercel/next.js/assets/53746400/58fa6200-1079-44ab-9624-b149ec6ca1ba)

**Expected Behavior**
Next 13.4.19 should maintain the same performance or lower than the Next 12.1.6

**To Reproduce**
Run request latency on Next 13.4.19 and Next 12.1.6 to see the differences

### Additional information

```js
Package version before upgrade:
react: 17.0.2
next: 12.1.6

Package version after upgrade: 
react: 18.2.0
next: 13.4.19
```

**Comment by macstr1k3r** [comment]
we're seeing the same thing.
